go/ir/derivation_test.go

```diff
@@ -223,6 +224,15 @@ func TestFeatureDerivation(t *testing.T) {
 
 	a := assert.New(t)
 	db := database.GetTestingDBSingleton()
+	var testDataSQL string
+	if testDB == "mysql" {
+		testDataSQL = testdata.FeatureDerivationCaseSQL
+	} else if testDB == "hive" {
+		testDataSQL = testdata.FeatureDerivationCaseSQLHive
+	}
+	if err := testdata.Popularize(db.DB, testDataSQL); err != nil {
```

Comment:
How about add the dataset inside https://github.com/sql-machine-learning/sqlflow/blob/ab1592cf3f998630540fde4ec7d23befb90b6812/go/database/testing.go#L86 and https://github.com/sql-machine-learning/sqlflow/blob/ab1592cf3f998630540fde4ec7d23befb90b6812/go/database/testing.go#L111?

Reply:
There are many places that create test datasets, will try refine this in later PRs.